src/vsr/replica.zig

```diff
@@ -4838,11 +4993,14 @@ pub fn ReplicaType(
             switch (message.header.command) {
                 .reserved => unreachable,
                 .request => {
+                    assert(!self.standby());
                     // Do not assert message.header.replica because we forward .request messages.
                     assert(self.status == .normal);
                     assert(message.header.view <= self.view);
                 },
                 .prepare => {
+                    maybe(self.standby());
+                    if (replica == self.replica) assert(self.primary());
```

Comment:
I _think_ this assert would be correct on main as well: only primary sends prepares to itself.